package.json

```diff
@@ -6,13 +6,20 @@
     "dependencies": {
         "@artsy/fresnel": "^3.2.1",
         "@deriv/deriv-api": "^1.0.11",
+        "@hookform/resolvers": "^3.0.1",
         "@livechat/customer-sdk": "^3.1.0",
         "@loadable/component": "^5.15.2",
+        "@radix-ui/react-accordion": "^1.1.0",
+        "@radix-ui/react-alert-dialog": "^1.0.3",
+        "@radix-ui/react-dropdown-menu": "^2.0.2",
         "@svgr/webpack": "^6.2.1",
         "@vercel/gatsby-plugin-vercel-builder": "^0.1.2",
+        "classnames": "^2.3.2",
```

Comment:
We don't use it anywhere, can we remove it?

Reply:
you fixed it thanks :pray: